src/Model/SearchData.php

```diff
@@ -275,6 +287,59 @@ public function setSelectedCreator(?string $selectedCreator): SearchData
         return $this;
     }
 
+    /**
+     * @return array|null
+     */
+    public function getStatuses(): ?array
```

Comment:
```suggestion
    public function getTodoStatuses(): ?array
```
Da wir in CommSy mit untersch. Status-Varianten zu tun haben, wäre es gut, diesen Status "Todo Status" zu benennen, denke ich.